resource/global/YoStarJP/resource/tasks.json

```diff
@@ -5852,6 +5852,34 @@
             ]
         ]
     },
+    "Phantom@Roguelike@NextLevel": {
+        "ocrReplace": [
+            [
+                "迷霧の岐路",
+                "迷雾歧途"
+            ],
+            [
+                "落陽の庭園",
+                "落阳庭院"
+            ],
+            [
+                "凋落の故地",
+                "故土残躯"
+            ],
+            [
+                "悪夢の幕開け",
+                "噩梦之始"
+            ],
+            [
+                "緋月の回廊",
+                "血月长廊"
+            ],
+            [
+                "渇欲の大広間",
+                "渴欲大厅"
```

Comment:
Is the ocrreplace needed?
b72ac8d7d307e02c0cd2e285ab7b3d37e3c0a0f9 this was for EN (KR and TXWY also only use "text" and not ocrreplace)